x-pack/plugins/security_solution/public/cloud_security_posture/components/vulnerabilities/vulnerabilities_preview.tsx

```diff
@@ -170,11 +80,71 @@ export const VulnerabilitiesPreview = ({ hostName }: { hostName: string }) => {
   const { CRITICAL = 0, HIGH = 0, MEDIUM = 0, LOW = 0, NONE = 0 } = data?.count || {};
 
   const totalVulnerabilities = CRITICAL + HIGH + MEDIUM + LOW + NONE;
+
+  const hasVulnerabilitiesFindings = totalVulnerabilities > 0;
+
```

Comment:
i noticed those checks are repeating multiple times throughout the pr, i suggest to export them to a utility function of some sort and reuse